README.md

```diff
@@ -20,23 +20,44 @@ Source code annotation tool offers an UI to annotate source code and review thes
 
 ## Installation
 
-## Github OAuth tokens
+### Environment Variables
 
-1. You need an OAuth application on GitHub. See [how to create OAuth applications on GitHub](https://developer.github.com/apps/building-oauth-apps/creating-an-oauth-app/).
+The next sections make use of several environment variables to configure the application. In this table you will find all of them grouped as a quick reference:
 
-In order to be able to use this application while running the tool locally, make sure you add http://127.0.0.1:8080/oauth-callback to the authorization callback URL field.
+| Variable | Required | Default value | Meaning |
+| -- | -- | -- | -- |
+| `CAT_JWT_SIGNING_KEY` | YES | - | Key used to sign JWT (JSON Web Tokens) in the server |
+| `CAT_OAUTH_CLIENT_ID` | YES | - | GitHub application [OAuth credentials](#github-oauth-tokens) |
+| `CAT_OAUTH_CLIENT_SECRET` | YES | - | GitHub application [OAuth credentials](#github-oauth-tokens) |
+| `CAT_OAUTH_RESTRICT_ACCESS` | | - | [Application access control](#access-control) based on GitHub groups or teams |
+| `CAT_OAUTH_RESTRICT_REQUESTER_ACCESS` | | - | [User role control](#access-control) based on GitHub groups or teams |
+| `CAT_HOST` | | `0.0.0.0` | IP address to bind the HTTP server |
+| `CAT_PORT` | | `8080` | Port address to bind the HTTP server |
+| `CAT_SERVER_URL` | | `<CAT_HOST>:<CAT_PORT>` | URL used to access the application (i.e. public hostname) |
+| `CAT_DB_CONNECTION` | | `sqlite:///var/code-annotation/internal.db` | Points to the internal application database. [Read below](#importing-and-exporting-data) for the complete syntax |
+| `CAT_EXPORTS_PATH` | | `./exports` | Folder where the SQLite files will be created when requested from `http://<your-hostname>/export` |
+| `CAT_ENV` | | `production` | Sets the log level. Use `dev` to enable debug log messages |
+| `CAT_STATIC_PROXY_TARGET` | | - | :exclamation: TODO: to be reviewed when [#145](https://github.com/src-d/code-annotation/pull/145) is closed |
```

Comment:
please remove it. #145 may never be merged in master or this variable will be replaced by something